addons/beehave/debug/tree_node.gd

```diff
@@ -4,7 +4,7 @@ extends RefCounted
 # Based on https://rachel53461.wordpress.com/2014/04/20/algorithm-for-drawing-trees/
 
 const sibling_distance: float = 20.0
-const tree_distance: float = 0
+# const tree_distance: float = 0
```

Comment:
Was never set and seemed to be used only in one place. Replaced with `sibling_distance`

Reply:
Any reason this is commented out? Can we just remove this instead?